doc/ref_model/chapters/chapter01.md

```diff
@@ -107,7 +107,7 @@ This section introduces the high-level principles of infrastructure abstraction
 This document focuses on the documenting the higher level concepts that are needed to identify **Reference Model**. **Figure 1-6** below highlights its scope in more details.
```

Comment:
Change the figure references in Text. Same for 1.4 and 1.7

Reply:
you have an eye of eagle :) I Love it!

since @markshostak is away for thanksgiving, I have fixed them on his behalf. please resolve and approve if you happy with the modification.

Reply:
Thanks @rabi-abdel. 😄

Reply:
> 
> 
> since @markshostak is away for thanksgiving, I have fixed them on his behalf. please resolve and approve if you happy with the modification.

Yeah, but American holidays aren't the quite the same as European holidays...